state_manage/actions/actions.ts

```diff
@@ -0,0 +1,11 @@
+import * as ActionTypes from './actionTypes';
+
+export const setName = (name: string) => ({
+    type: ActionTypes.SET_NAME,
+    name
```

Comment:
I didn't know you can have something other than payload lol! But I think it's better to do `payload: name` because the redux action convention is type + payload. Or even better, you can do `payload: { name }` and then in reducer you can do `{ name } = action.payload`

Reply:
Not entirely sure what benefit the latter would provide. I get changing it to payload. I'll release fixes and would you be able to comment why I should do payload: { name }?

Reply:
So if you just do `payload: name`, then your payload is a string, but if you do `payload: { name }` then the payload is an object containing a field `name` with value being the string. 

The benefit of latter might be a little subjective as it is mainly a matter of style/convention, but I think it's more clear because it kinda allows you to describe what you're keeping. If instead of `name` you have a string literal `'Bob'`, then it's clearer to do: `payload: { name: 'bob' }` then `payload: 'bob'`. Also a lot of times we want to have multiple things in the payload, so having a dictionary format of key value pairs makes it easy.

Reply:
Got it makes sense